doc/admin-guide/files/remap.config.en.rst

```diff
@@ -518,20 +514,95 @@ mapping rules. (It is activated before any mappings and is never
 deactivated.) The filter `local_only` will only be applied to the
 second mapping.
 
-Implict IPAllow filter
-======================
+Special Filter and ip_allow Named Filter
+----------------------------------------
+
+If :file:`ip_allow.yaml` has a "deny all" filter, it is treated as a special filter that is applied before remapping for
+optimizaion. To control this for specific remap rules, a named filter called `ip_allow` is pre-defined. This named filter is
+activated implicitly in default. To stop applying the special rule, disable the `ip_allow` filter as shown below.
```

Comment:
Should that be "by default" ?

Reply:
Do you mean "deactivate" the ip_allow named filter by default? 

A tricky behavior is if this is deactivated, step 1 and 4 is disabled. Should we disable only step 1  by default?

> 1. Special "deny all" filter in :file:`ip_allow.yaml`
> 2. In-line Filter in :file:`remap.config`
> 3. Named Filter in :file:`remap.config`
> 4. Filters in :file:`ip_allow.yaml`